### PR TITLE
[New UX] Student settings styling

### PR DESCRIPTION
**Pull request by mgolawski**
Screenshot 2019-09-27 14 39 20

### CODE REVIEW

File: src/screens/studentSettings/PlanDisplayPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    alignItems: 'center',
+    width: 350,
+    zIndex: 1,
+  },
+  beneathPlanCard: {
+    ...getElevation(3),
+    height: 30,
+    width: 320,
+    backgroundColor: palette.background,
+    translateY: -22,
```
Comment (maciejhein) [inline src/screens/studentSettings/PlanDisplayPreview.tsx]: Why are you using `translateY` ? Maybe you can replace it by `position` relative, absolute.

File: src/components/FixedValueSlider.tsx [hunk 2]
```diff
@@ -39,18 +36,15 @@ export class FixedValueSlider extends React.PureComponent<Props, State> {
   setIndex = (index: number) => this.setState({ index });
 
   render() {
-    const { iconLeft, iconRight, options } = this.props;
+    const { options } = this.props;
```
Comment (maciejhein) [inline src/components/FixedValueSlider.tsx]: empty line

**Comment by bklukaczewski** [comment]
Need to confirm with UX design if we really want to remove the icons at slider left and right ends.
It would be best to send Rafał before and after versions.

**Comment by mgolawski** [comment]
This is how it looks like in mockups
https://xd.adobe.com/spec/01d5ebeb-d8a4-4a08-7f23-309931817948-77e4/screen/2b3c4fb3-421a-4e8c-8404-5b7ce5f46609/Ustawienia-podopiecznego

**Pull request merged**